apps/ledger-live-desktop/src/renderer/families/avalanchepchain/Delegation/Row.jsx

```diff
@@ -0,0 +1,96 @@
+// @flow
+import React from "react";
+import { getAccountUnit } from "@ledgerhq/live-common/account/index";
+import { formatCurrencyUnit } from "@ledgerhq/live-common/currencies/index";
+import type { Account } from "@ledgerhq/live-common/lib/types";
+import { BigNumber } from "bignumber.js";
+import styled from "styled-components";
+import { TableLine } from "./Header";
+import { isDefaultValidatorNode } from "@ledgerhq/live-common/families/avalanchepchain/utils";
+import FirstLetterIcon from "~/renderer/components/FirstLetterIcon";
+import Box from "~/renderer/components/Box";
+import { IconContainer } from "~/renderer/components/Delegation/ValidatorRow";
+import Logo from "~/renderer/icons/Logo";
+import type { AvalancheDelegation } from "@ledgerhq/live-common/families/avalanchepchain/types";
+import moment from "moment";
+import type { ThemedComponent } from "~/renderer/styles/StyleProvider";
+import { getInitialLanguageLocale } from "~/renderer/reducers/settings";
+
+type Props = {
+  account: Account,
+  delegation: AvalancheDelegation,
+  onManageAction: (delegation: AvalancheDelegation, action: string) => void,
+  onExternalLink: (address: string) => void,
+};
+
+export function Row({ delegation, account, onExternalLink }: Props) {
+  const onExternalLinkClick = () => onExternalLink(delegation.txID);
+
+  const locale = getInitialLanguageLocale();
+  moment.locale(locale);
```

Comment:
I think this doesn't belong here. especially not at each time we "re-renderer". what was the problem to solve? it sounds like it's a more general bug?

Reply:
iirc, it's because we do not know which locale format we want to apply in live, so we were expecting to use the computer one.